things.yml

```diff
@@ -127,3 +127,7 @@ things:
     - number: 2
     - image: https://cdn.shopify.com/s/files/1/0748/6277/products/spring_loaded_glitter_bomb_multi_1200x.jpg?v=1512349319
     - link: https://www.ruindays.com/products/spring-loaded-glitter-bomb?variant=12317569155143
+  six-cup-chemex:
```

Comment:
How pretty! Let's just call it `six-cup-coffeemaker` since us mortals don't know what a six cup chemex is!

Reply:
The ?v is also probably some sort of visitor / tracking, so you can remove it from both links, for example:

https://cdn.shopify.com/s/files/1/0056/4562/products/chemex-at-home-web_large.jpg

Reply:
thanks! new commit created with the suggested changes.